patches/server/0891-allow-setting-of-arrowpickup.patch

```diff
@@ -0,0 +1,63 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Aaron <78169748+tl0x@users.noreply.github.com>
+Date: Fri, 8 Apr 2022 17:03:45 -0400
+Subject: [PATCH] allow setting of arrowpickup
+
+
+diff --git a/src/main/java/net/minecraft/world/entity/projectile/AbstractArrow.java b/src/main/java/net/minecraft/world/entity/projectile/AbstractArrow.java
+index 53d0024daf6963ac4dab575666b0d6a74a39a958..00fd2849e4dd31b506645dbb9bd7128a453ac638 100644
+--- a/src/main/java/net/minecraft/world/entity/projectile/AbstractArrow.java
++++ b/src/main/java/net/minecraft/world/entity/projectile/AbstractArrow.java
+@@ -603,6 +603,8 @@ public abstract class AbstractArrow extends Projectile {
+ 
+     public abstract ItemStack getPickupItem();
+ 
++    public abstract void setPickupItem(ItemStack stack);
```

Comment:
I think the field should be here on AbstractArrow and then you can have a pickup item override for all things that extend AbstractArrow.